scripts/vercel-ignore-preview.sh

```
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Skip Preview builds unless PR has label "preview"
+if [ "$VERCEL_ENV" = "preview" ]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)